sdk-tests/src/test/java/io/dapr/it/binding/http/BindingIT.java

```diff
@@ -123,5 +123,6 @@ public void inputOutputBinding() throws Exception {
       }
       assertEquals("hello", resultClass.message);
     }, 8000);
+    client.close();
```

Comment:
Use try-with-resources instead of closing it explicitly.

Reply:
Change it everywhere.

Reply:
done